src/core/lib/event_engine/tcp_socket_utils.cc

```diff
@@ -224,10 +312,15 @@ absl::optional<int> ResolvedAddressIsWildcard(
 absl::StatusOr<std::string> ResolvedAddressToNormalizedString(
     const EventEngine::ResolvedAddress& resolved_addr) {
   EventEngine::ResolvedAddress addr_normalized;
-  if (ResolvedAddressIsV4Mapped(resolved_addr, &addr_normalized)) {
-    return ResolvedAddressToString(addr_normalized);
+  if (!ResolvedAddressIsV4Mapped(resolved_addr, &addr_normalized)) {
+    addr_normalized = resolved_addr;
+  }
+  auto scheme = GetScheme(addr_normalized);
+  GRPC_RETURN_IF_ERROR(scheme.status());
+  if (*scheme == "unix") {
```

Comment:
Now that `ResolvedAddressToString` will return a unix path for unix sockets, I think this scheme checking is redundant.

Reply:
Fixed.